stats/cloud/api.go

```diff
@@ -112,3 +116,28 @@ func (c *Client) TestFinished(referenceID string, thresholds ThresholdResult, ta
 	err = c.Do(req, nil)
 	return err
 }
+
+func (c *Client) Login(email string, password string) (*LoginResponse, error) {
+	url := fmt.Sprintf("%s/login", c.baseURL)
+
+	data := struct {
+		Email    string `json:"email"`
+		Password string `json:"password"`
+	}{
+		email,
+		password,
+	}
+
+	req, err := c.NewRequest("POST", url, data)
+	if err != nil {
+		return nil, err
+	}
+
+	lr := LoginResponse{}
+	err = c.Do(req, &lr)
+	if err != nil {
+		return nil, errors.New("Failed to login.")
```

Comment:
@ppcano Isn't this going to mask the error response from the server? Wouldn't it be better to just forward/return the `err` instead of creating a new one that hides the underlying problem.

Related but perhaps to consider separately, we should handle rate-limit error responses. We don't have any checks for 429 status code in `checkResponse` (https://github.com/loadimpact/k6/blob/master/stats/cloud/client.go#L156).

Reply:
@robingustafsson This is a temporary solution until we handle better the error message from the server (including the limitations for the `rate`limits). This will be handled as a separate task.

Reply:
No masking error messages.